hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/CryptoDeleteHandlerTest.java

```diff
@@ -384,6 +392,19 @@ void failsIfEitherDeleteOrTransferAccountDoesntExist() throws PreCheckException
                 .has(responseCode(ACCOUNT_ID_DOES_NOT_EXIST));
     }
 
+    @Test
+    @DisplayName("check that fees are 1 for delete account trx")
+    void testCalculateFeesInvocations() {
```

Comment:
nit: suggested rename to clearly indicate purpose 
`testCalculateFeesReturnsCorrectFeeForDeleteAccount()`

Reply:
addressed, ty